src/sample/sample.vcxproj.user

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup />
+</Project>
```

Comment:
I can't put this comment below for some reason, but you need to remove the .obj and .pdb files below from the working tree. (`git rm \<path\>`)